src/requested-data/requested-data-reducer.ts

```diff
@@ -51,3 +51,12 @@ export function requestedData(
             return state;
     }
 }
+
+function checkStateLoaded(state: RequestedDataState, action: RequestedDataAction): boolean {
```

Comment:
Codacy found an issue: [All 'checkStateLoaded' signatures should be adjacent](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5948151)